lib/commons/dom/get-rect-stack.js

```diff
@@ -1,12 +1,25 @@
 import visuallySort from './visually-sort';
 import { getRectCenter } from '../math';
 
-export function getRectStack(grid, rect, recursed = false) {
+export function getRectStack(
+  grid,
+  rect,
+  recursed = false,
```

Comment:
default arguments should be at the end - there must not be any mandatory arguments after default arguments